static/src/javascripts/projects/common/modules/ui/message.js

```diff
@@ -143,9 +159,28 @@ class Message {
         );
     }
 
+    trapFocus(): void {
+        const messageEl = this.$siteMessage[0];
+        const [trapStartEl, trapEndEl] = new Array(2).fill(
```

Comment:
is this effectively doing

```js
const trapStartEl = document.createElement('div');
const trapEndEl = document.createElement('div');
```
?

Reply:
i think so? 

i wouldn't be afraid of repeating yourself here – if you write out explicitly what is happening, the next developer that reads your code should be able to understand your intention immediately. 

as things stand, you need to understand that an Array is being created, filled and destructured (none of which is the specific intention of the code) to understand that you're creating two divs and assigning them to variables (which I think is your intention). 

it's also worth bearing in mind [how this code will be handled by babel](http://babeljs.io/repl/#?babili=false&browsers=%0A&build=&builtIns=usage&code_lz=MYewdgzgLgBA2lATgQwA4GUrMVAogGwBoYk1cwATAgXRgF4YwBTAdxgEFEUBPACgCYAlADoAZgEt8-XgCgAkBRDAArgFsmYKMOCImyKEwJN1m3gHIK4gG5nBMwQG4gA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=true&fileSize=true&lineWrap=true&presets=es2015%2Ces2017%2Creact%2Cstage-0%2Cstage-1%2Cstage-2%2Cstage-3&prettier=true&targets=&version=6.26.0&envVersion=1.6.2) (85 bytes becomes 1kb)

Reply:
holy moly that babel size!